cinema/views.py

```diff
@@ -34,6 +38,33 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
+    @staticmethod
+    def _params_to_ints(qs):
+        return [int(some_id) for some_id in qs.split(",")]
```

Comment:
Move this method outside of the ViewSet, because you're duplicating it in the another Viewset